cmd/install/install_test.go

```diff
@@ -0,0 +1,58 @@
+package install
+
+import (
+	. "github.com/onsi/gomega"
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	"testing"
+)
+
+func TestOptions_Validate(t *testing.T) {
+	tests := map[string]struct {
+		inputOptions Options
+		expectError  bool
+	}{
+		"aws private platform without private creds and region errors": {
+			inputOptions: Options{
+				PrivatePlatform: string(hyperv1.AWSPlatform),
+			},
+			expectError: true,
+		},
+		"invalid private platform errors": {
```

Comment:
why is this us case failing, can't PrivatePlatform be empty?

Reply:
Private Platform is dictated currently to be either None or AWS (It cannot be empty). None is effectively empty

Reply:
I don't think that has to be this way but any changes there I would like done on a future pr